### DISCUSSION TITLE
Is mixed-SEI model (diffusion + reaction limited) available on PyBaMM?

### DISCUSSION THREAD

**Discussion by ganmad** [pybamm-team/PyBaMM#3320] [Q&A] [answered]
Hello Team,

I am wondering whether mixed-SEI model is available on PyBaMM. We have diffusion limited and reaction limited models available individually. Is the mixed model available to capture SoC, Temperature dependence, decelerating capacity fade due to SEI all together accurately? The mixed model can be represented as this equation:

![image](https://github.com/pybamm-team/PyBaMM/assets/50149439/ece6fe3f-c976-4f7d-8bd0-b9c9b0a98784)

Best regards,
Ganesh

**Comment by ganmad** [comment, marked as answer]
After rearranging the terms in the above equation, I realised that "ec reaction limited" indeed captures the above mixed-mode controlled SEI. Thanks for providing this. 

Best regards,

**Comment by fsenoner** [comment]
Hello @ganmad,

Have you tried to adjust the parameters for the diffusion model (D_SEI) and the reaction-limited model (k_SEI) so that one part of the equation becomes insignificant? You would get a equation which models only the diffusion or only the reaction limited part. I believe that in this scenario, either D_SEI >> k_SEI or vice versa should hold true to simplify the equation.

Best regards
Fabian

**Reply by ganmad** [reply]
Hi Fabian, 

Yes, you are right. I tried tweaking those parameters for making it either diffusion controlled or reaction control. It's working fine for me.

Best regards,
Ganesh

**Reply by fsenoner** [reply]
Hi Ganesh,
thank you for your response.

Would it be possible for you to share a plot of your results or the code?
I tried to adjust the parameters in the OKane2022 parameter set and these are my results
![image](https://github.com/pybamm-team/PyBaMM/assets/130640867/8d3ba764-116f-4868-ad2c-56e503051f79)

Best regards,
Fabian

**Reply by ganmad** [reply]
Its working fine for me.

Diffusion limited with default values of OKane2022:
![image](https://github.com/pybamm-team/PyBaMM/assets/50149439/59426de8-4ae1-4253-a02c-80cee819d6a5)

Reaction limited with updated values:
 `"EC diffusivity [m2.s-1]": 2.5e-22*1000,
        "SEI kinetic rate constant [m.s-1]": 1e-12*0.001,  `
![image](https://github.com/pybamm-team/PyBaMM/assets/50149439/8b992617-0be8-473d-b87f-af469ee613e0)

In your simulations, try reducing k_SEI further and increasing D_EC. Regards.
